contracts/integrations/test/fuzz_tests/pool_membership_test.ts

```diff
@@ -0,0 +1,89 @@
+import { blockchainTests, constants } from '@0x/contracts-test-utils';
+import * as _ from 'lodash';
+
+import { Maker } from '../framework/actors/maker';
+import { PoolMember } from '../framework/actors/pool_member';
+import { AssertionResult } from '../framework/assertions/function_assertion';
+import { BlockchainBalanceStore } from '../framework/balances/blockchain_balance_store';
+import { DeploymentManager } from '../framework/deployment_manager';
+import { Simulation, SimulationEnvironment } from '../framework/simulation';
+
+import { PoolManagementSimulation } from './pool_management_test';
+import { StakeManagementSimulation } from './stake_management_test';
+
+class PoolMembershipSimulation extends Simulation {
+    protected async *_assertionGenerator(): AsyncIterableIterator<AssertionResult | void> {
+        const { deployment } = this.environment;
+
+        const poolManagement = new PoolManagementSimulation(this.environment);
+        const stakeManagement = new StakeManagementSimulation(this.environment);
```

Comment:
I don't think we need this here, since stake shouldn't really affect joining pools